lib/routes/achievement.js

```diff
@@ -38,7 +42,7 @@ router
       }}
     } else {
       await models.db.achievement
-        .findOrCreate({where: {sequence, name, distance, title, subtitle, description, media}})
+        .findOrCreate({where: {sequence: sequence, name: name, distance: distance, description: description, icon_name: iconName, map_image: mapImage, title: title, subtitle: subtitle, content: content, media: media}})
```

Comment:
I think for the dictionary syntax you only need to specify key *and* value for icon_name and map_image since their names differ
```suggestion
        .findOrCreate({where: {sequence, name, distance, description, icon_name: iconName, map_image: mapImage, title, subtitle, content, media}})
```